
{'date': 'Mon Jun 27 2022 17:42:07.343 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    Aventura().main()
AttributeError: 'Aventura' object has no attribute 'main'
'''},
{'date': 'Mon Jun 27 2022 17:42:53.297 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    print(ADV[:100])
NameError: name 'ADV' is not defined
'''},
{'date': 'Mon Jun 27 2022 17:59:45.694 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 332
    Aventura().main(ADV)
  module <module> line 28
    i = "\n".join( ini for kind, ini in self.roteiro if kind == "I")
ValueError: need more than 1 value to unpack
'''},
{'date': 'Mon Jun 27 2022 18:14:27.910 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  if kind == "L"
                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jun 27 2022 18:14:39.743 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 340
    Aventura().main(ADV)
  module <module> line 36
    locais[umlocal] = [cmd]
IndexError: list assignment index out of range
'''},
{'date': 'Mon Jun 27 2022 18:16:27.618 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 341
    Aventura().main(ADV)
  module <module> line 38
    locais[umlocal].append(cmd)
IndexError: list index out of range
'''},
{'date': 'Mon Jun 27 2022 18:38:23.831 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 344
    Aventura().main(ADV)
  module <module> line 34
    print([lc[o] for lc in locais])
NameError: name 'o' is not defined
'''},
{'date': 'Mon Jun 27 2022 21:52:25.84 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 20
  _, cmd = adv[0]]
                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jun 27 2022 21:52:36.135 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 21
  loc, desc = cmd.split(":)
                           ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Jun 27 2022 21:52:47.975 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 352
    Aventura().main(ADV)
  module <module> line 40
    locais = [Cenario(rot[ini:fim]) for ini,fim in zip(lro, lro[1:]+[len(rot)])]
  module <module> line 22
    Cenario.CENA[loc] = self
AttributeError: 'Cenario' object has no attribute 'CENA'
'''},
{'date': 'Mon Jun 27 2022 21:53:15.267 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 352
    Aventura().main(ADV)
  module <module> line 42
    print([lc[0] for lc in locais])
AttributeError: 'Cenario' object has no attribute '__getitem__'
'''},
{'date': 'Mon Jun 27 2022 22:12:23.274 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 362
    Aventura().main(ADV)
  module <module> line 49
    locais = [Cenario(rot[ini:fim]) for ini,fim in zip(lro, lro[1:]+[len(rot)])]
  module <module> line 26
    self.objeto = [Objeto(rot[ini:fim]) for ini,fim in zip(lro, lro[1:]+[len(adv)])]
  module <module> line 35
    loc, self.descreve = cmd.split(":")
ValueError: need more than 1 value to unpack
'''},
{'date': 'Mon Jun 27 2022 23:05:33.903 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 384
    Aventura().main(ADV)
  module <module> line 71
    locais = [Cenario(rot[ini:fim]) for ini,fim in zip(lro, lro[1:]+[len(rot)])]
  module <module> line 28
    self.objeto = [Objeto(adv[ini:fim],self) for ini,fim in zip(lro, lro[1:]+[len(adv)])]
  module <module> line 41
    self.verbo = {adv[ini].split(":")[-2]:Verbo(adv[ini:fim], cenario) for ini,fim in zip(lro, lro[1:]+[len(adv)])}
AttributeError: 'list' object has no attribute 'split'
'''},
{'date': 'Mon Jun 27 2022 23:14:09.27 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 43
  self.verbo = {(verbo:=Verbo(adv[ini:fim], cenario)).dercreve:verbo
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jun 27 2022 23:14:36.945 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 43
  self.verbo = {(verbo:=Verbo(adv[ini:fim], cenario)).descreve:verbo
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Jun 27 2022 23:17:00.936 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''[VOCE ESTA NA COZINHA DA VOVO AQUI VOCE VAI PREPARAR A SALADA, VOCE ESTA NA SALA DE JANTAR. COM UMA MESA E MOVEIS DE VIME, O SAGUAO DA VOVO E GRANDE, OS ARCOS LEVAM A SALA DE JANTAR E A  ESCADA AO SEGUNDO ANDAR, ESTE E O TERRENO DA TIA LOURDES, VOCE SE EQUILIBRA NO MURO ETA MURO ALTO SO! MELHOR EU ME  AGARRAR EM ALGUMA COISA!  PARECE FIRME PARA SE AGARRAR  E TEM UM MAMAO AMARELINHO NELE, VOCE ESTA NA ORLA DA PISCINA, VOCE ESTA EM FRENTE DA CASA DO SEU ZE, O CASEIRO, VOCE ESTA NA RAMPA QUE LEVA DO PATIO ATE A PISCINA, NO PATIO HA UMA JAQUEIRA,A CASA DA VO E BEM EM FRENTE, UMA RAMPA  LEVA 'A PISCINA 'A DIREITA, VOCE ESTA NA RUA SAPHIRO ARAUJO,A NO SITIO DA VOVO..., VOCE ESTA EM SEU APARTAMENTO NA CIDADE, LOUCA PARA IR AO SITIO]
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 389
    Aventura().main(ADV)
  module <module> line 79
    print(Cenario.OBJ)
  module <module> line 46
    return f"@{self.descreve} <{list(self.verbo.keys())}>"
  module <module> line 1
    (list(self.verbo.keys()))
AttributeError: 'list' object has no attribute 'keys'
'''},
{'date': 'Tue Jun 28 2022 09:43:02.164 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 386
    Aventura().main(ADV)
  module <module> line 81
    locais.pop().vai()
  module <module> line 31
    texto = self.descreve + "\nVocê pode ver" + "\n".join(ob for ob in self.objeto)
TypeError: sequence item 0: expected str instance, Objeto found
'''},
{'date': 'Tue Jun 28 2022 09:43:02.829 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 386
    Aventura().main(ADV)
  module <module> line 81
    locais.pop().vai()
  module <module> line 31
    texto = self.descreve + "\nVocê pode ver" + "\n".join(ob for ob in self.objeto)
TypeError: sequence item 0: expected str instance, Objeto found
'''},
{'date': 'Tue Jun 28 2022 10:00:55.24 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 391
    Aventura().main(ADV)
TypeError: __init__() missing 2 positional arguments: adv,cenario
'''},
{'date': 'Tue Jun 28 2022 10:13:43.463 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 393
    Aventura().main(ADV)
  module <module> line 88
    locais.pop().vai()
  module <module> line 33
    texto = self.descreve + "\nVocê pode ver:\n" + "\n".join(ob.mostra() for ob in self.objeto.keys())
AttributeError: 'str' object has no attribute 'mostra'
'''},